lib/cinch/plugins/wordgames/game.rb

```diff
@@ -35,14 +32,25 @@ def number_of_guesses_phrase
     end
 
   protected
-    def guess_correct?(word, response)
-      if @word == word
-        response.game_won
-        true
+    def blank?(bound)
+      bound == Blank_str
+    end
+
+    def evaluate_guess!(word)
+      if !@dict.word_valid?(word) && !@ref_dict.word_valid?(word)
+        :not_a_word
+      elsif @solution == word
+        :correct
       else
-        response.wrong_word(@word.before_or_after(word), word)
-        false
+        if @solution.before?(word)
+          @upper_bound = word if blank?(@upper_bound) || word < @upper_bound
+          :missed_south
+        else
+          @lower_bound = word if blank?(@lower_bound) || word > @lower_bound
+          :missed_north
```

Comment:
I'm loving your use of "identifying symbols." I've never used this before but it looks really handy